common-components/schemas/type/decimal.md

```diff
@@ -0,0 +1,37 @@
+# Decimal
+
+The `Decimal` type represents a decimal numeric value. It uses a non-floating
+point representation in order to avoid precision errors.
+
+## Schema
+
+A `Decimal` represents a decimal number in a form similar to scientific
+notation.
+
+It has two fields:
+
+- The `digits` field is a 64-bit integer representing the significant digits of
+  the number.
+- The `exponent` field is a 32-bit integer represesnting the position of the
+  deicmal point within the value of the `digits` field; it is the power of ten
+  to which `digits` should be raised.
+
+  When this value is unset or `0`; in that case, the value of the `Decimal` is
+  simply the value of `digits`.
+
+  When the exponent is greater than 0, represents the number of trailing zeroes
```

Comment:
```suggestion
  When the exponent is greater than 0, `exponent` represents the number of trailing zeroes
```

Reply:
Is the way I originally wrote this not idiomatic, i.e. implicitly "[this field] represents"?